test/org/sagebionetworks/bridge/dynamodb/DynamoHealthDataDaoDdbTest.java

```diff
@@ -1,30 +1,40 @@
 package org.sagebionetworks.bridge.dynamodb;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
 
 import javax.annotation.Resource;
 
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
 import com.fasterxml.jackson.databind.JsonNode;
 import org.joda.time.LocalDate;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.sagebionetworks.bridge.TestUtils;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.healthdata.HealthDataRecord;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import org.sagebionetworks.bridge.TestConstants;
 import org.sagebionetworks.bridge.dao.ParticipantOption;
 import org.sagebionetworks.bridge.json.BridgeObjectMapper;
 
+import java.io.IOException;
+import java.util.List;
+
 @ContextConfiguration("classpath:test-context.xml")
 @RunWith(SpringJUnit4ClassRunner.class)
 public class DynamoHealthDataDaoDdbTest {
     private static final long CREATED_ON = 1424136378727L;
-    private static final String HEALTH_CODE = "health-code";
-    private static final String RECORD_ID = "record-id";
+    private static final String HEALTH_CODE = TestUtils.randomName(DynamoHealthDataDaoDdbTest.class);
+    private static final String RECORD_ID = TestUtils.randomName(DynamoHealthDataDaoDdbTest.class);
     private static final String DATA_TEXT = "{\"data\":\"dummy value\"}";
     private static final String METADATA_TEXT = "{\"metadata\":\"dummy meta value\"}";
-    private static final String SCHEMA_ID = "schema-id";
+    private static final String SCHEMA_ID = TestUtils.randomName(DynamoHealthDataDaoDdbTest.class);
```

Comment:
Randomizing the schema ID is probably not necessary, since nothing here uses it as a key.